tests/quantization/test_precision_init.py

```diff
@@ -728,28 +715,7 @@ def _get_config_path(self):
 
 
 MANUAL_CONFIG_TEST_PARAMS = [
-    ManualSampleConfigTestParams(name="mobilenet_v2_imagenet_mixed_int_manual.json",
-                                 bit_stats=[['8', '23.077', '23.932', '47.009'],
-                                            ['4', '22.222', '30.769', '52.991']]),
-    ManualSampleConfigTestParams(name="mobilenet_v2_imagenet_mixed_int_manual_staged.json",
-                                 bit_stats=[['8', '23.077', '24.786', '47.863'],
-                                            ['4', '22.222', '29.915', '52.137']]),
-    ManualSampleConfigTestParams(name="resnet50_imagenet_mixed_int_manual.json",
-                                 bit_stats=[['8', '21.600', '23.200', '44.800'],
-                                            ['4', '21.600', '33.600', '55.200']]),
-    ManualSampleConfigTestParams(name="resnet50_imagenet_mixed_int_manual_staged.json",
-                                 bit_stats=[['8', '21.600', '28.000', '49.600'],
-                                            ['4', '21.600', '28.800', '50.400']]),
-    ManualSampleConfigTestParams(name="squeezenet1_1_imagenet_mixed_int_manual.json",
-                                 bit_stats=[['8', '24.528', '30.189', '54.717'],
-                                            ['4', '24.528', '20.755', '45.283']]),
-    ManualSampleConfigTestParams(name="squeezenet1_1_imagenet_mixed_int_manual_staged.json",
-                                 bit_stats=[['8', '24.528', '30.189', '54.717'],
-                                            ['4', '24.528', '20.755', '45.283']]),
-    ManualTestConfigTestParams(name='resnet18_cifar10_mixed_int_manual.json',
-                               bit_stats=[['8', '27.907', '25.581', '53.488'],
-                                          ['2', '18.605', '0', '18.605'],
-                                          ['4', '2.326', '25.581', '27.907']])
```

Comment:
@ljaljushkin please update the manual configs in a separate PR so that they use the propagation-based mode and so that I don't need to remove them altogether in this PR.

Reply:
ultimately, we won't use manual configs, but serialized quantizer setup

Reply:
@ljaljushkin so are you ok with me removing the configs and tests in this PR?

Reply:
decided to update the manual configs, as we probably would need similar scenario even with `quantizer_setup`
#564